docs/source/test_network.md

```diff
@@ -0,0 +1,594 @@
+# Using the Fabric test network
+
+After you have downloaded the Hyperledger Fabric Docker images and samples, you
+can deploy a test network by using scripts that are provided in the
+`fabric-samples` repository. You can use the test network to learn about Fabric
+by running nodes on your local machine. More experienced developers can use the
+network to test their smart contracts and applications. The network is meant to
+be used only as a tool for education and testing. It should not be used as a
+template for deploying a production network.
+
+The sample network deploys a Fabric network with Docker Compose. Because the
+nodes are isolated within a Docker Compose network, you cannot connect other
+running Fabric nodes to the test network.
```

Comment:
I would rather be even stronger in terms of telling folks that this is not a prod deployment, and not to try to use it to connect to other nodes. How about this:
`Because the
nodes are isolated within a Docker Compose network, the test network is not
configured to connect to other running fabric nodes.
`